diff --git a/debian/compat b/debian/compat
index f599e28..ec63514 100644
--- a/debian/compat
+++ b/debian/compat
@@ -1 +1 @@
-10
+9
diff --git a/debian/control b/debian/control
index 1ea744d..ba15689 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: silversearcher-ag
 Section: utils
 Priority: optional
 Maintainer: Hajime Mizuno <mizuno-as@ubuntu.com>
-Build-Depends: debhelper (>= 10), pkg-config, libpcre3-dev, zlib1g-dev, liblzma-dev, python-cram, git
+Build-Depends: debhelper (>= 9), dh-autoreconf, pkg-config, libpcre3-dev, zlib1g-dev, liblzma-dev, python-cram, git
 Standards-Version: 4.1.1
 Homepage: https://github.com/ggreer/the_silver_searcher
 
diff --git a/debian/rules b/debian/rules
index 49ac1a5..95d9d81 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,6 +1,14 @@
 #!/usr/bin/make -f
 %:
-	dh $@
+	dh $@ --with autoreconf
+
+override_dh_autoreconf:
+	dh_autoreconf --as-needed
+
+override_dh_auto_configure:
+	dh_auto_configure -- --disable-silent-rules
+
+override_dh_auto_test:
 
 override_dh_install:
 	mv  $(CURDIR)/debian/silversearcher-ag/usr/share/the_silver_searcher/completions/ag.bashcomp.sh \
@@ -9,3 +17,4 @@ override_dh_install:
 	mv  $(CURDIR)/debian/silversearcher-ag/usr/share/zsh/site-functions/_the_silver_searcher \
             $(CURDIR)/debian/silversearcher-ag/usr/share/zsh/vendor-completions/
 	rm -rf $(CURDIR)/debian/silversearcher-ag/usr/share/zsh/site-functions
+
